client/src/panels/panel_declarations/finances/transfer_payments/tp_by_region.js

```diff
@@ -11,81 +11,130 @@ import {
   declarative_charts,
 } from "../../shared.js";
 import { Canada } from '../../../../charts/canada/index.js';
+import { SlideToggle, SpinnerWrapper } from '../../../../components/index.js';
+import { get_static_url, make_request } from '../../../../request_utils.js';
 
 const { std_years } = year_templates;
+
 const formatter = formats["compact2_raw"];
 
 const { text_maker, TM } = create_text_maker_component(text);
 const { provinces, provinces_with_article } = businessConstants;
 const { A11YTable } = declarative_charts;
 
-const calculate_common = (subject, table) => {
-  const level = subject.level;
 
-  if ( level === 'dept' && !_.has(table.depts , subject.id) ){
-    return false;
+function loadPopulation(){
+  const parse_csv_string = csv_string => _.tail( d3.csvParseRows( _.trim(csv_string) ) );
+
+  const load_csv = () => make_request( get_static_url(`csv/population.csv`) )
+    .then( csv_string => parse_csv_string(csv_string) );
+    
+  return load_csv().then(function(loaded) {
+    const transformed = _.chain(loaded)
+      .keyBy(row => row[0])
+      .mapValues(row => _.chain(row)
+        .tail()
+        .map(value => parseInt(value))
+        .value()
+      )
+      .value();
+    
+    return transformed;
```

Comment:
I don't think declaring the result of the chain separate from it's return here has any value. Might just make people reading the code thing that some additional step has gone missing. If the new variable you were declaring had a more specific name describing the output of some long lodash chain, then declaring it separately from returning it would be a good documentation strategy.

Maybe with with the variable name `population_values_by_prov_code` instead of just `transformed`?

Reply:
Changed variable name to population_values_by_prov_code
Done